Typing errors when using with Electron + Angular project

I got these typing error when using this library + Angular15 + Electron22 

# Error Message

```
Error: node_modules/@u4/opencv4nodejs/typings/FacemarkAAMParams.d.ts:33:7 - error TS2304: Cannot find name 'Point2'.

33   s0: Point2[]; // was string
         ~~~~~~

Error: node_modules/@u4/opencv4nodejs/typings/TrackerBoostingParams.d.ts:1:47 - error TS2552: Cannot find name 'TrackerBoostingParamsI'. Did you mean 'TrackerBoostingParams'?

1 export class TrackerBoostingParams implements TrackerBoostingParamsI {
                                                ~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@u4/opencv4nodejs/typings/TrackerBoostingParams.d.ts:1:14
    1 export class TrackerBoostingParams implements TrackerBoostingParamsI {
                   ~~~~~~~~~~~~~~~~~~~~~
    'TrackerBoostingParams' is declared here.
```

The error is gone after I added an import to FacemarkAAMParams and remove implements from TrackerBoostingParams.

is this bugs? should I do a pull request?

here is my package.json

```json
{
  "name": "angular-electron",
  "version": "11.2.0",
  "description": "Angular 14 with Electron 21 (Typescript + SASS + Hot Reload)",
  "homepage": "https://github.com/maximegris/angular-electron",
  "author": {
    "name": "Maxime GRIS",
    "email": "maxime.gris@gmail.com"
  },
  "keywords": [
    "angular",
    "angular 14",
    "electron",
    "electron 19",
    "nodejs",
    "typescript",
    "playwright",
    "eslint",
    "sass",
    "windows",
    "mac",
    "linux"
  ],
  "main": "app/main.js",
  "private": true,
  "scripts": {
    "postinstall": "electron-builder install-app-deps",
    "ng": "ng",
    "start": "npm-run-all -p electron:serve ng:serve",
    "ng:serve": "ng serve -c web -o",
    "build": "npm run electron:serve-tsc && ng build --base-href ./",
    "build:dev": "npm run build -- -c dev",
    "build:prod": "npm run build -- -c production",
    "web:build": "npm run build -- -c web-production",
    "electron": "electron",
    "electron:serve-tsc": "tsc -p tsconfig.serve.json",
    "electron:serve": "wait-on tcp:4200 && npm run electron:serve-tsc && electron . --serve",
    "electron:local": "npm run build:prod && electron .",
    "electron:build": "npm run build:prod && electron-builder build --publish=never",
    "electron-rebuild": "build-opencv --electron --version 4.7.0 --buildRoot C:/opencv build",
    "test": "ng test --watch=false",
    "test:watch": "ng test",
    "e2e": "npm run build:prod && playwright test -c e2e/playwright.config.ts e2e/",
    "e2e:show-trace": "playwright show-trace e2e/tracing/trace.zip",
    "version": "conventional-changelog -i CHANGELOG.md -s -r 0 && git add CHANGELOG.md",
    "lint": "ng lint"
  },
  "dependencies": {
    "@angular/common": "15.1.2",
    "@angular/compiler": "15.1.2",
    "@angular/core": "15.1.2",
    "@angular/forms": "15.1.2",
    "@angular/language-service": "15.1.2",
    "@angular/platform-browser": "15.1.2",
    "@angular/platform-browser-dynamic": "15.1.2",
    "@angular/router": "15.1.2",
    "@u4/opencv4nodejs": "6.4.1",
    "rxjs": "7.5.7",
    "tslib": "^2.4.0",
    "zone.js": "~0.11.8"
  },
  "devDependencies": {
    "@angular-builders/custom-webpack": "15.0.0",
    "@angular-devkit/build-angular": "15.1.3",
    "@angular-eslint/builder": "15.2.0",
    "@angular-eslint/eslint-plugin": "15.2.0",
    "@angular-eslint/eslint-plugin-template": "15.2.0",
    "@angular-eslint/schematics": "15.2.0",
    "@angular-eslint/template-parser": "15.2.0",
    "@angular/cli": "15.1.3",
    "@angular/compiler-cli": "15.1.2",
    "@electron/rebuild": "3.2.10",
    "@ngx-translate/core": "14.0.0",
    "@ngx-translate/http-loader": "7.0.0",
    "@playwright/test": "1.27.1",
    "@types/jasmine": "4.3.0",
    "@types/jasminewd2": "2.0.10",
    "@types/node": "18.11.0",
    "@typescript-eslint/eslint-plugin": "^5.43.0",
    "@typescript-eslint/parser": "^5.43.0",
    "conventional-changelog-cli": "2.2.2",
    "electron": "22.0.3",
    "electron-builder": "23.6.0",
    "electron-debug": "3.2.0",
    "electron-reloader": "1.2.3",
    "eslint": "^8.28.0",
    "eslint-plugin-import": "2.26.0",
    "eslint-plugin-jsdoc": "39.3.6",
    "eslint-plugin-prefer-arrow": "1.2.3",
    "jasmine-core": "4.4.0",
    "jasmine-spec-reporter": "7.0.0",
    "karma": "6.4.1",
    "karma-coverage-istanbul-reporter": "3.0.3",
    "karma-electron": "7.3.0",
    "karma-jasmine": "5.1.0",
    "karma-jasmine-html-reporter": "2.0.0",
    "node-polyfill-webpack-plugin": "2.0.1",
    "npm-run-all": "4.1.5",
    "playwright": "1.27.1",
    "ts-node": "10.9.1",
    "typescript": "4.8.4",
    "wait-on": "6.0.1",
    "webdriver-manager": "12.1.8"
  },
  "engines": {
    "node": ">= 14.15.0"
  },
  "browserslist": [
    "chrome 100"
  ]
}
```

here is my tsconfig

```json
{
  "compileOnSave": false,
  "compilerOptions": {
    "outDir": "./dist/out-tsc",
    "module": "es2020",
    "sourceMap": true,
    "declaration": false,
    "moduleResolution": "node",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowJs": true,
    "target": "ES2022",
    "typeRoots": [
      "node_modules/@types"
    ],
    "lib": [
      "es2017",
      "es2016",
      "es2015",
      "dom"
    ],
    "useDefineForClassFields": false
  },
  "exclude": [
    "node_modules"
  ],
  "angularCompilerOptions": {
    "strictTemplates": true,
    "fullTemplateTypeCheck": true,
    "annotateForClosureCompiler": true,
    "strictInjectionParameters": true,
    "skipTemplateCodegen": false,
    "preserveWhitespaces": true,
    "skipMetadataEmit": false,
    "disableTypeScriptVersionCheck": true
  }
}

```

--- a/typings/FacemarkAAMParams.d.ts
+++ b/typings/FacemarkAAMParams.d.ts
@@ -1,4 +1,5 @@
 import { Facemark } from "./Facemark";
+import { Point2 } from "./Point2";
 
 export class FacemarkAAM extends Facemark { }
 

--- a/typings/TrackerBoostingParams.d.ts
+++ b/typings/TrackerBoostingParams.d.ts
@@ -1,4 +1,4 @@
-export class TrackerBoostingParams implements TrackerBoostingParamsI {
+export class TrackerBoostingParams {
   readonly numClassifiers: number;
   readonly samplerOverlap: number;
   readonly samplerSearchFactor: number;

